docs/user/access-control.md

```diff
@@ -120,6 +120,9 @@ kubectl create clusterrolebinding example-kubeapps-service-catalog-admin --clust
 
 In order to list the configured App Repositories in Kubeapps, create and apply the
 `kubeapps-repositories-read` Role in the namespace Kubeapps is installed in.
+If you installed kubeapps with the helm-chart, this role will be present in the target
```

Comment:
In our case, using the chart is the only way of getting Kubeapps so what I would say is that a role named `kubeapps-repositories-read` already exists and it's necessary to get the different repositories installed. Then you can remove the line below that applies the manifest from the URL

Reply:
BTW, now that I take a look again to the docs, we can also create the role `*-repositories-write` since the use case is very similar to this one (and update the docs).

Reply:
There are a number of references in the doc to manifest files to create resources with kubectl - I guess all should go then?